common/db.go

```diff
@@ -423,6 +423,21 @@ func (db *DB) SelectOrchs(filter *DBOrchFilter) ([]*DBOrch, error) {
 	return orchs, nil
 }
 
+func (db *DB) OrchCount(filter *DBOrchFilter) (int, error) {
+	if db == nil {
+		return 0, nil
+	}
+
+	row := db.dbh.QueryRow(buildOrchCountQuery((filter)))
```

Comment:
1.  Since `buildOrchCountQuery()` returns `string, error` we should check if there is an error returned and we should also only pass in the query string to `QueryRow()`.

2. Can we remove the parens around `filter` when passing it into `buildOrchCountQuery()`?

Reply:
fixed in dfd3cc53